components/block/BlockDetails.tsx

```diff
@@ -251,6 +249,33 @@ const BlockDetails = () => {
             </Typography>
             <Divider variant="middle" className={classes.divider} />
           </Grid>
+
+          <Grid item xs={12} >
+            <Typography variant="body2" component="h2" gutterBottom>
+              Signers
+            </Typography>
+            {data.block && data.block.signers
+              ?
+              data.block.signers.map((row: any, index: number) => {
+                return (
+                  <span key={index}>
+
+                    <Typography variant="body2" gutterBottom >
+                      <Link
+                        href="/validatorGroup/[validatorGroupDetails]/"
+                        as={row.validator && row.validator.validatorGroup && row.validator.validatorGroup.address ?
+                          `/validatorGroup/${row.validator.validatorGroup.address}` : ""}
```

Comment:
I wonder if we should have an individual page for each validator

Reply:
I have created a separate issue for this  - we can track it here https://github.com/forbole/big-dipper-celo-client/issues/62